Mathlib/Data/List/Basic.lean

```diff
@@ -33,6 +32,12 @@ universe u v w
 
 variable {ι : Type*} {α : Type u} {β : Type v} {γ : Type w} {l₁ l₂ : List α}
 
+/-- `≤` implies not `>` for lists. -/
+@[deprecated (since := "2024-07-27")]
```

Comment:
Should this say what to use instead?

Reply:
It's just `rfl`, and it seems no one uses it, and it dates from before the port. 🤷‍♀️